awx/ui/client/test/e2e/README.md

```diff
@@ -0,0 +1,21 @@
+## AWX E2E
```

Comment:
It's implied with the `docker exec` command below, but might be worth explicitly mentioning the API server must be up and running on `https://localhost:8043` (if not overridden) and what can be overridden with the environment variables referenced in the settings file.